src/main/java/com/example/home_work_4/StringTotal.java

```diff
@@ -0,0 +1,393 @@
+package com.example.home_work_4;
+
+import java.util.Scanner;
+
+public class StringTotal {
+
+    public static void main(String[] args) {
```

Comment:
Please remove method main, as it is not needed to test your solution here. You were supposed to test it on [https://codingbat.com/java](url)